src/pages/[scheduleId]/edit.tsx

```diff
@@ -69,6 +69,21 @@ export default function Create(props: Props) {
     }
   }
 
+  const deleteHandler = async () => {
+    const { query } = router
+    const user_id = Number(query.id)
+    const schedule = props.schedule
+    const users: User[] = schedule.users || [{ name: '', memo: '', availables: null }]
+    try {
+      await deleteUser(props.id, users[user_id])
+      await router.push('/' + props.id)
```

Comment:
したみたいにすると、指定したdynamic routeのqueryに対して正確な値を入れることができるので、ただの文字列の足し算から生まれるバグとか曖昧さを消すことができる、おすすめ
```suggestion
      await router.push({
            pathname: '/[scheduleId]', 
            query: {
                  scheduleId: props.id
            }
      }
)
```